tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -150,13 +150,19 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
     //        NSBinarySearchingOptions* searchForBoth = NSBinarySearchingFirstEqual | NSBinarySearchingLastEqual
     //        auto _ = objectIndexInArray(array, 9, 0, 13, searchForBoth)
 
-    auto notFound = objectIndexInArray(array, 11, 0, 13);
-    ASSERT_EQ_MSG(notFound, NSNotFound, @"NSArray return NSNotFound if object is not found.");
+    ASSERT_EQ_MSG(objectIndexInArray(array, 11, 0, array.count),
```

Comment:
Good catch. 

---
In reply to: [67916770](https://github.com/Microsoft/WinObjC/pull/569#discussion_r67916770) [](ancestors = 67916770)